cmd/node/node_test.go

```diff
@@ -566,6 +546,8 @@ func TestSpacemeshApp_NodeService(t *testing.T) {
 		poetHarness.HTTPPoetClient, clock, h, edSgn)
 	require.NoError(t, err)
 
+	appCtx, appCancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
I'd suggest not using an explicit short timeout as it tends to create flaky tests on slower runners. The test timeout is handled by `go test -timeout <value>` externally. IMO there's no need to duplicate it in test.
```suggestion
	appCtx, appCancel := context.WithCancel(context.Background())
```